### PR TITLE
Create module and method docstrings for ENTSOE

### PR DESCRIPTION
**Pull request by systemcatch**
Let me know if any need changing.

### CODE REVIEW

File: parsers/ENTSOE.py [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+Production
+Exchanges
+Exchange Forecast
+Day-ahead Price
+Generation Forecast
+Consumption Forecast
+
+The zones below use methods from this parser.
+
+['AT', 'BA', 'BE', 'BG', 'CH', 'CZ', 'DE', 'DK', 'DK-DK1', 'DK-DK2', 'EE', 'ES',
```
Comment (corradio) [inline parsers/ENTSOE.py]: Can we leave this out? The info is already in zones.json

Comment (systemcatch) [reply parsers/ENTSOE.py]: The list or the whole module docstring?

Comment (corradio) [reply parsers/ENTSOE.py]: just the list (I prefer having only one place to maintain instead of two).
Good initiative by the way!

Comment (systemcatch) [reply parsers/ENTSOE.py]: Thanks, I'll change it.

File: parsers/ENTSOE.py [hunk 2]
```diff
@@ -133,7 +127,7 @@ def check_response(response, function_name):
 def query_ENTSOE(session, params, now=None, span=[-24, 24]):
     """
     Makes a standard query to the ENTSOE API with a modifiable set of parameters.
-    Allows an existing session to be passed. Default timespan of +/- 24 hours.
+    Allows an existing session to be passed. 7Default timespan of +/- 24 hours.
```
Comment (corradio) [inline parsers/ENTSOE.py]: typo?

Comment (systemcatch) [reply parsers/ENTSOE.py]: indeed

**Pull request merged**